src/hooks/proposal/useProposalsCoreData.ts

```diff
@@ -8,7 +8,14 @@ import { UserContext } from 'context/UserContextProvider';
 import { ProposalsFilter, ProposalView, UserRole } from 'generated/sdk';
 import { useDataApi } from 'hooks/common/useDataApi';
 
-export function useProposalsCoreData(filter: ProposalsFilter) {
+export function useProposalsCoreData(
+  filter: ProposalsFilter,
+  first?: number,
```

Comment:
I see in a few places you have the same set of parameters passed around. I wonder if it is not easier to define a typed object and pass that around. 

```
first?: number,
offset?: number,
sortField?: string,
sortDirection?: string,
searchText?: string
```

Reply:
Thanks, I could use the query object directly to be more succinct 👍